addon/components/search/filters.hbs

```diff
@@ -11,39 +11,45 @@
       Form.submit
       type="submit"
       label=(t "ember-gwr.components.modelForm.save")
-      disabled=(or this.isSaving @disabled this.isSubmitBtnDisabled)
-      loading=this.submit.isRunning
+      disabled=(or this.isSaving @disabled this.isSubmitDisabled)
+      loading=@onSubmit.isRunning
       buttonComponent=(component "model-form/submit" color="primary")
     )
     as |Submit|
   }}
-  {{!-- Error Select Component. Allows to filter by "hasError" --}}
   {{#let
```

Comment:
Move both definitions in one let

Reply:
done

Reply:
Did you push your latest changes? still looks the same.